README.md

```diff
@@ -14,54 +16,59 @@ Run `rails g async_request`
 
 ### Usage
 
-Simply call `async_request(your_worker, *args)` like this:
+This gem provides a  that will call an `execute` a method of any class you give and will store the response in the database. In order to do this, we provide a model named `AsyncRequest::Job` that will handle the queue of the worker with the params.
```

Comment:
of any INSTANCE of any given class, right ? they are not class methods, they are instance methods